exodus-installer.sh

```diff
@@ -52,19 +57,46 @@ exodus_download() {
 }
 
 
+# Download and check the exodus package to verify
+# SHA hash
+exodus_verify_hashes() {
+  #
+  # If JP's key doesn't exist...
+  if ! gpg -k | grep -q "JP Richardson" ; then 
+    jpKey='https://keybase.io/jprichardson/pgp_keys.asc?fingerprint=12408650e2192febe4e7024c9d959455325b781a'
+    # ...Import JP Richardson's Public Key
+    curl -s $jpKey | gpg --import -q
+  fi
+
+  local HASHES=`exodus_download_url hashes-exodus${EDEN_DOWNLOAD_INFIX}-$1.txt`
+  curl -s $HASHES | gpg --verify
+  if ! [ $? -eq 0 ]; then
+    return 1
+  fi
+  from_hash=`curl -s $HASHES | grep linux | perl -lane 'print $F[0]'`
+  to_hash=`shasum -a 256 $2 | perl -lane 'print $F[0]'`
```

Comment:
perl? Is it everywhere? Maybe we will use awk/sed? I think cut all before space is not hard with them.
Out of interest why `shasum -a 256` instead `sha256sum`?

Reply:
I used `perl` anyway because `shasum` is in the `perl` package. It was just one less package to make sure you have on your system (although I think most of the time, `awk` is there).

Again, `shasum -a 256` was a direct paste from Konnor's existing script for verifying checksums.

Reply:
But at least `sha256sum` is in `coreutils` under Ubuntu, something you are likely to already have installed! :-)